src/pretix/control/forms/global_settings.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, *args, **kwargs):
             )),
             ('opencagedata_apikey', SecretKeySettingsField(
                 required=False,
-                label=_("OpenCage API key for geocoding"),
+                label=_("OpenCage API key for geo   ding"),
```

Comment:
Please revert this change, it is unrelated, and wrong.

Reply:
ok